2.0.0 changes

cs-abhishek-shukla opened the pull request:
- Typographical, grammatical, and information flow issues
- Created a new configuring ingestion playbooks section
- Added known issue to release notes
- Streamlined release notes
- Modified simulation and production mode
- Multiple other changes

cs-dev-chaudhary commented on docs/usage.md:
2nd Asset name is : Stratix Services Router-4121 in line number 39. 

Unable to comment on the line in github.

cs-dev-chaudhary commented on docs/usage.md:
```diff
@@ -17,7 +17,7 @@ Refer to the section **OT - Vulnerability Management** to understand how this so
-            Before using the simulation mode, make the following configurations are in place:
+            Before using the simulation mode, make the following changes:
             <ol>
                 <li>To use actual data of listed CVEs, configure the <strong><em>NIST National Vulnerability Database</em></strong> connector.</li>
```
Please change "To use actual data of listed CVEs" to "To get latest report of listed CVEs"

cs-ankit-gupta commented on docs/usage.md:
```diff
@@ -157,51 +176,57 @@ The simulation mode has some sample data that helps you get a better understandi
 ## Production Mode
 
-1. Set global variable `Demo_mode` to *`true`*. Refer to the section [Global Variables](https://docs.fortinet.com/document/fortisoar/7.4.3/playbooks-guide/488685/dynamic-values#Global_Variables) in the **Playbooks Guide** of the platform documentation for more information.</li>
+1. Set global variable `Demo_mode` to *`false`*. Refer to the section [Global Variables](https://docs.fortinet.com/document/fortisoar/7.4.3/playbooks-guide/488685/dynamic-values#Global_Variables) in the **Playbooks Guide** of the platform documentation for more information.</li>
```
On line number 168 add two points:
1. Does Patch Exist?</strong> as <strong><em>Unknown</em>.
2. Reset the scenario before moving to production mode.
@cs-abhishek-shukla

cs-abhishek-shukla committed:
2.0.0 changes
- Typographical, grammatical, and information flow issues
- Created a new configuring ingestion playbooks section
- Added known issue to release notes
- Streamlined release notes
- Modified simulation and production mode
- Multiple other changes
